pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -528,10 +526,8 @@ func (h *NetworkUtilsHandler) DisableTXOffloadChecksum(ifaceName string) error {
 var DHCPServer = dhcp.SingleClientDHCPServer
 var DHCPv6Server = dhcpv6.SingleClientDHCPv6Server
 
-func initHandler() {
-	if Handler == nil {
-		Handler = &NetworkUtilsHandler{}
-	}
+func NewHandler() NetworkHandler {
```

Comment:
[Returning an interface instead of the concrete](https://ehaas.net/blog/the-go-interface/#return-an-interface) is usually not the right thing to do.
I would suggest returning the concrete struct and let the caller assign it to an interface variable.